README.md

```diff
@@ -27,6 +98,104 @@ def App():
 
 run(App)
 ```
+HTML with idiom example
+
+```python
+from idom import html
```

Comment:
These examples are copy-pasted from file in `docs/` which implies that these examples are already documented elsewhere and don't necessarily need to be surfaced here.